src/components/Layouts/ReportLayout.js

```diff
@@ -24,13 +24,17 @@ const ReportLayout = ({ sections, headerLeftImage, headerRightImage }) => {
         Object
           .keys(sections)
           .map((rowPos) =>
-            <div className="report-row" key={rowPos} style={sections[rowPos].style}>
+            <div
+              className={ sections[rowPos][0].class ? 'report-row ' +sections[rowPos][0].class : 'report-row'}
```

Comment:
Just to be on the safe side, can you make sure `sections[rowPos]` is not empty before addressing the first index?

Reply:
can we have a row without any columns? I didn't se any option to define rows without at least one column

Reply:
This is fine then.
You will still have lint errors, I would fix it and merge.

Reply:
Thanks!